dpctl/tensor/_dlpack.pyx

```diff
@@ -965,41 +965,66 @@ cdef object _create_device(object device, object dl_device):
 def from_dlpack(x, /, *, device=None, copy=None):
```

Comment:
Would it be helpful to add a test to cover such scenario? (input numpy.ndarray, no `device` keyword passed)?
It seems I failed to find it.

Reply:
It can be found in `test_generic_container_legacy_np`, on line 796:

https://github.com/IntelPython/dpctl/blob/631d4c3bb52db0961b98ce3283c9819d36382f85/dpctl/tests/test_usm_ndarray_dlpack.py#L792-L799